internal/rest/resources/cluster.go

```
@@ -613,10 +614,9 @@ func clusterMemberDelete(s state.State, r *http.Request) response.Response {
 	err = s.Database().Transaction(r.Context(), func(ctx context.Context, tx *sql.Tx) error {
 		return cluster.DeleteCoreClusterMember(ctx, tx, remote.Address.String())
 	})
-	if err != nil {
+	if err != nil && !force {
```

For `truststore` and `internalClient.ResetClusterMember` below we already ignore error if `force` is set. I add this here to support the case where the database is cleaned up correctly but the truststore etc not yet. (Related to #487)

---

Makes sense to me thanks. Please move it into its own commit for clarity.

---

Done